apps/server/lib/lexical/server/code_intelligence/completion/translations/callable.ex

```diff
@@ -118,11 +128,35 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.Callable do
 
   @default_functions ["module_info", "behaviour_info"]
 
-  defp maybe_boost(item, %_{name: name}, default_boost \\ 5) do
-    if String.starts_with?(name, "__") or name in @default_functions do
-      item
-    else
-      Builder.boost(item, default_boost)
+  defp maybe_boost(item, callable, %Env{position_module: position_module}) do
+    %_{
+      name: name,
+      origin: origin,
+      metadata: metadata
+    } = callable
+
+    # elixir_sense suggests child_spec as a callback, though it's not formally one.
+    deprecated? = Map.has_key?(metadata, :deprecated)
+    dunder? = String.starts_with?(name, "__")
+    callback? = callback?(callable)
+
+    local_priority =
+      cond do
+        dunder? -> 9
+        callback? -> 8
+        true -> 1
+      end
+
+    cond do
+      origin === "Kernel" or origin === "Kernel.SpecialForms" or name in @default_functions ->
```

Comment:
what if you've imported a module or function? Will it be local?

Reply:
Completions from imports are sorted as remote for now, so they're below local functions, macros, and callbacks, but before globally available callables.

I mostly did this out of convenience, and that the results seemed serviceable. I found it was easy to identify global and local functions, and have other callable completions default to remote.

I could see a case for imported stuff being local, but I think the results look nice now.

Reply:
I think this sort order is good. I was worried they'd be considered local. 
My thoughts here have been "make the closest things have higher priority", and this does that.